lib/services.rb

```diff
@@ -6,4 +6,8 @@ def self.content_store
   def self.rate_limiter
     @rate_limiter ||= Ratelimit.new("deliveries")
   end
+
+  def self.worldwide_api
```

Comment:
Instead of needing to define this here you can use `GdsApi.worldwide` wherever you need to use the API:

https://github.com/alphagov/gds-api-adapters/blob/52d5d97e9b5822462bb533944666b02a1596bed1/lib/gds_api.rb#L206-L208

Reply:
Thanks @thomasleese, changed.